### ISSUE TITLE
[Q]: Build using Release Scheme?

### ISSUE THREAD

**Issue by wkoutre** [joshuapinter/apphubdeploy#9]
Is there a flag or some way to build with a Release XCode scheme/configuration?

**Comment by joshuapinter** [comment]
Great question. It's been a while since I've dug in deep here. My guess is that it does that by default. AppHub is about delivering Release JS Bundles over the air so it wouldn't make sense for it to create debug JS Bundles.

It would be great if you could confirm, though.

**Comment by wkoutre** [comment]
It wouldn't make sense, but it's happening! At least to me.

All OTA bundles I've created have all my debugging-flagged features turned on, unfortunately.

Most of my flags are based off `process.env.NODE_ENV`, though, and not `__DEV__`, so it may just be a product of that.

I also use the `transform-remove-console` Babel plugin on production builds and.. logs are definitely still there if I run from an OTA bundle.

**Comment by wkoutre** [comment]
I think I've narrowed it down.

Passing the flag `-t "debug"` targets "debugging enabled" users, but also builds under the debugging config.

Passing with `-t "all"` builds with a Release config.

I was attempting to use debug AppHub distributions for segmenting "test users" when rolling out new features, but I needed it to be a Release config.

I'll just use versioning instead; good to know, though! Closing.
